Gemfile.lock

```diff
@@ -37,22 +38,23 @@ GEM
       rspec-core (~> 3.9.0)
       rspec-expectations (~> 3.9.0)
       rspec-mocks (~> 3.9.0)
-    rspec-core (3.9.0)
-      rspec-support (~> 3.9.0)
-    rspec-expectations (3.9.0)
+    rspec-core (3.9.2)
+      rspec-support (~> 3.9.3)
+    rspec-expectations (3.9.2)
```

Comment:
### [rspec-expectations](https://github.com/rspec/rspec-expectations)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.9.0 → 3.9.2

[_[change-log](https://github.com/rspec/rspec-expectations/blob/v3.9.2/Changelog.md), [source-code](https://github.com/rspec/rspec-expectations)_]





<details>
<summary>Commits</summary>

A change of **16** commits. See the full changes on [the compare page](https://github.com/rspec/rspec-expectations/compare/v3.9.0...v3.9.2).

These are the first 10 commits:
- (234aeae) [Updates from rspec-dev (2019-12-26) (#1149)](https://github.com/rspec/rspec-expectations/commit/234aeaeb3d916f00d1c496eb0b63a386ed54a02f)
- (bc409c6) [Updated common plaintext files (from rspec-dev) [ci skip] (#…](https://github.com/rspec/rspec-expectations/commit/bc409c6d4da88484f57825ed6ee96c45cffc2f06)
- (2c9338c) [Updatted travis build scripts (from rspec-dev)](https://github.com/rspec/rspec-expectations/commit/2c9338cfbb7796b6ee074dade2e6cb9693d32cfd)
- (6fff617) [Handle broken method handles in `respond_to` and `have_attri…](https://github.com/rspec/rspec-expectations/commit/6fff617bee4d74f74adfa98da33677a8dbbd8243)
- (9b9d276) [Bump yard to support Ruby 2.7](https://github.com/rspec/rspec-expectations/commit/9b9d276a0d4b62194c18e702f3ba13b588bd5e81)
- (908a5f8) [Updated travis build scripts (from rspec-dev) (#1147)](https://github.com/rspec/rspec-expectations/commit/908a5f8292fb060ecd025646dcecfeb4fe70275f)
- (d671f88) [Version 3.9.1](https://github.com/rspec/rspec-expectations/commit/d671f8854532168b5e57c0479b5e6e5a61bc6cc8)
- (84fc254) [Updated travis build scripts (from rspec-dev)](https://github.com/rspec/rspec-expectations/commit/84fc25417da95bb59a72677888d469d037684e8d)
- (123f359) [Update rake dependency to avoid security warning](https://github.com/rspec/rspec-expectations/commit/123f3590163ca6bf6f9610599169049ce4c720e5)
- (96bad96) [yield_control hardening (#1167)](https://github.com/rspec/rspec-expectations/commit/96bad969dc56e75b64661bc29f01207b46aaa30c)


</details>